tests/test_reverse_urls_are_supported.py

```
@@ -0,0 +1,43 @@
+from django.urls import reverse
```

Ruff (0.8.2)

Missing docstring in public module

(D100)